src/patternfly/components/ClipboardCopy/clipboard-copy.scss

```
@@ -59,6 +63,7 @@
 
 .#{$clipboard-copy}__group {
   display: flex;
+  gap: var(--pf-t--global--spacer--xs);
```

I think this should be a component var

```suggestion
  gap: var(--#{$clipboard-copy}__group--Gap);
```

with `--#{$clipboard-copy}__group--Gap: var(--pf-t--global--spacer--xs);` in the component vars